src/main/resources/application.yaml

```diff
@@ -71,6 +70,30 @@ spring:
       out-of-order: true
       ignore-missing-migrations: true
 
+  flyway.enabled: ${ENABLE_DB_MIGRATE:true}
+  flyway.locations: classpath:db/migration
+#  datasource:
+#    driver-class-name: org.postgresql.Driver
+#    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}${DB_OPTIONS:}
+#    username: ${DB_USER_NAME}
+#    password: ${DB_PASSWORD}
+#    properties:
+#      charSet: UTF-8
+#    hikari:
+#      minimumIdle: 2
```

Comment:
should we have all the commented out lines?

Reply:
Removed.